package.json

```diff
@@ -14,29 +14,31 @@
   "author": "Mux <help@mux.com> (https://mux.com/)",
   "main": "src/mux.js",
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
This bumps the min node version to 10.

Node 8 has reached end-of-life: https://github.com/nodejs/Release#end-of-life-releases

<img width="967" alt="node-8-end-of-life_2020-05-29_14-44-23" src="https://user-images.githubusercontent.com/764988/83308291-5c6fab80-a1bb-11ea-8ef4-3ee626e8c8eb.png">


Unfortunately, this will require a major version bump but I think we should still do it. The reason this change is part of this PR is because the `eslint` version bump below (required to lint typescript stuff) requires node 10+

As @JacquiManzi remembered -- some [kook 2 years ago before they even worked at Mux needed Node 8 support](https://github.com/muxinc/mux-node-sdk/pull/4)... Ah how things come full circle 😄